From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rene Hickersberger <r@renehsz.com>
Date: Sat, 24 Jun 2023 01:25:27 +0200
Subject: [PATCH] Disable broken features

Mouse support is currently broken and X11 does not make sense on Serenity.
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 955965aad67967a5acd73822647b2ce11d4a819e..6e1f0476c761a8f7a8d53e2c4f0fe714a20198a6 100644
--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,7 @@ O_DEBUG := 0  # debug binary
 O_NORL := 0  # no readline support
 O_PCRE := 0  # link with PCRE library
 O_NOLC := 0  # no locale support
-O_NOMOUSE := 0  # no mouse support
+O_NOMOUSE := 1  # no mouse support
 O_NOBATCH := 0  # no built-in batch renamer
 O_NOFIFO := 0  # no FIFO previewer support
 O_CTX8 := 0  # enable 8 contexts
@@ -26,7 +26,7 @@ O_QSORT := 0  # use Alexey Tourbin's QSORT implementation
 O_BENCH := 0  # benchmark mode (stops at first user input)
 O_NOSSN := 0  # disable session support
 O_NOUG := 0  # disable user, group name in status bar
-O_NOX11 := 0  # disable X11 integration
+O_NOX11 := 1  # disable X11 integration
 O_MATCHFLTR := 0  # allow filters without matches
 O_NOSORT := 0  # disable sorting entries on dir load
 
